essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-use-case-software-engineer.md

```diff
@@ -1,2 +1,102 @@
+
+![project plann](https://user-images.githubusercontent.com/89600478/235844017-2f00d682-5ddd-4fb5-a935-0055d0f3bb34.jpg)
+<div align="center">
+  <p><small style="font-size: 70%"><em>Project planning page</em></small></p>
+</div>
+
+## Development Tasks:
+Once I have my project plan in place, it's time to start working on the development tasks. To manage my development tasks, I create a new list for each task and add notes and tags as needed. For example, I might create a note for the task description, a note for any subtasks, and a note for the estimated time to complete. By tagging each note with the project name and task name, I can easily find all the notes related to that task.
+
+
+<div align="center">
+  <img src="https://user-images.githubusercontent.com/89600478/235844308-d90c9446-732f-4b0a-8808-f7ace7f3e774.png">
+</div>
+
+
+<div align="center">
+  <p><small style="font-size: 70%"><em>Yes I use boards to design my Database schema</em></small></p>
+</div>
+
+ 
+## Bugs and Issues
+Of course, no software project is without its bugs and issues. That's where Appflowy's bug tracking features come in. To track bugs and issues, I create a new list for each issue and add notes and tags as needed. For example, I might create a note for the bug description, a note for the steps to reproduce, and a note for the priority level. By tagging each note with the project name and issue name, I can easily find all the notes related to that issue.
+
+
+
+
+## Project Completion
+Once all the development tasks are completed and all the bugs are fixed, it's time to wrap up the project. To manage the project completion, I create a new list for the project and add notes and tags as needed. For example, I might create a note for the final deliverables, a note for any outstanding issues, and a note for the project status. By tagging each note with the project name, I can easily find all the notes related to that project.
+
+
+## Other Exciting features for Software Engineers
+## Open AI Integration
+Being a Software Engineer you must be tired of reading through long, technical documents and articles. Well, Appflowy's got your back with their new OpenAI integration. With just a few clicks, you can now quickly summarize lengthy texts and extract key information using the power of OpenAI.
+All you have to do is highlight the text you want to summarize and click on the 'Summarize' option in Appflowy's menu. And that's not it, it can fix spellings for you too.
+Before you get started, make sure to enter your OpenAI API key in the settings menu. This feature can be a huge time-saver for busy software engineers who need to stay up-to-date on the latest industry news and trends. So what are you waiting for? Give it a try and make your life easy.
+
+  ![image (8)](https://user-images.githubusercontent.com/89600478/236600222-7f6b30f0-2e39-4f5d-a642-7a90948ea624.png)
```

Comment:
Could you try to insert a video or Gif file to demonstrate the AI feature you mentioned earlier?